modules/ROOT/examples/live-demos/advtemplate-mce-clipboard/index.html

```diff
@@ -0,0 +1,29 @@
+<textarea id="advanced-template-mce-clipboard">
+  <h3>Using the mce-clipboard Marker with Advanced Templates<h3>
+  <h4>To insert a template</h4>
+  <div>
+    <ol>
+      <li>
+        Select <strong>Template…</strong> from the <strong>Insert</strong> menu or<br />
+        select the <strong>Insert template</strong> toolbar button.
+      </li>
+      <li>
+      Select the template to add to the TinyMCE document from the <strong>Templates</strong> dialog that presents.
+        <ol>
+          <li>
+          Click the <strong>Without an mce-clipboard marker</strong> category to see and select a template that <strong>does not</strong> use the <code>{{mce-clipboard}}</code> marker.
+          </li>
+          <li>
+          Click the <strong>With an mce-clipboard marker</strong> category to see and select a template that <strong>does</strong> use the <code>{{mce-clipboard}}</code> marker.
+          </li>
+        </ol>
+      </li>
+      <li>
+        click <strong>Insert</strong> or press <strong>Return</strong>.
+      </li>
+    </ol>
+  </div>
+  <h4>Noting the difference</h4>
+  <p>The <em>Paste your clipboard</em> template without the <code>{{mce-clipboard}}</code> marker, does not replace the and content marker within the template text.</p>
+  <p>By contrast, the <em>Paste your clipboard</em> template with the <code>{{mce-clipboard}}</code> marker replaces the marker with content copied to the users native clipboard.</p>
```

Comment:
Using the two categories with the templates having the same title `Paste your clipboard` looks a bit confusing. Why don't you use just two templates with the names: "Without an mce-clipboard marker" and "With an mce-clipboard marker" and do away with categories?

Reply:
@vpyshnenko updated demo, as suggested- but will need to re-test before going live to ensure the demo works.

Reply:
You forgut to update the line 27 and 28 according the new templates structure
```suggestion
  <p>The <em>Without an mce-clipboard marker</em> template does not replace the and content marker within the template text.</p>
  <p>By contrast, the <em>With an mce-clipboard marker</em> template replaces the marker with content copied to the users native clipboard.</p>
```